vlo-web-app/src/main/java/eu/clarin/cmdi/vlo/wicket/pages/FacetedSearchPage.java

```diff
@@ -262,7 +269,7 @@ protected void onAjaxSearchPagination(AjaxRequestTarget target) {
     }
 
     private SearchResultsHeaderPanel createResultsHeader(String id, IModel<QueryFacetsSelection> model, AbstractPageableView<SolrDocumentExpansionPair> resultsView, IDataProvider<SolrDocument> solrDocumentProvider, IModel<Long> recordCountModel) {
-        return new SearchResultsHeaderPanel(id, model, resultsView, solrDocumentProvider, recordCountModel) {
+    	return new SearchResultsHeaderPanel(id, model, resultsView, solrDocumentProvider, recordCountModel) {
```

Comment:
please make sure your formatting options are set according with the existing practice (i.e. configure to use spaces instead of tabs)